crates/rnote-ui/src/dialogs/export.rs

```diff
@@ -578,32 +600,15 @@ fn create_filedialog_export_doc_pages(
 
     filedialog.set_initial_folder(get_initial_folder_for_export(appwindow, canvas).as_ref());
 
+    // does it work with mimetype but not the previous thing ?
+
     let filter = FileFilter::new();
     // We always need to be able to select folders
     filter.add_mime_type("inode/directory");
-    match doc_pages_export_prefs.export_format {
-        DocPagesExportFormat::Svg => {
-            filter.add_mime_type("image/svg+xml");
-            filter.add_suffix("svg");
-            filter.set_name(Some(&gettext("Svg")));
-        }
-        DocPagesExportFormat::Png => {
-            filter.add_mime_type("image/png");
-            filter.add_suffix("png");
-            filter.set_name(Some(&gettext("Png")));
-        }
-        DocPagesExportFormat::Jpeg => {
-            filter.add_mime_type("image/jpeg");
-            filter.add_suffix("jpg");
-            filter.add_suffix("jpeg");
-            filter.set_name(Some(&gettext("Jpeg")));
-        }
-    }
-
-    let filter_list = gio::ListStore::new::<FileFilter>();
-    filter_list.append(&filter);
-    filedialog.set_filters(Some(&filter_list));
-
+    // note : we revert https://github.com/flxzt/rnote/pull/1075
```

Comment:
I think some got accidentally deleted here?

Reply:
Tell me if I'm wrong but for me this opens a folder chooser, so file filters aren't used

Reply:
Right, I think my reasoning when implementing this was that all files that could possibly be overwritten by the export would be visible. Picking fails when not a folder is selected so it should be fine to add these the files filter

Reply:
Ok, I'll try to put the code back and adapt it (I think I need some more `cfg` directives to make it work)